static/429.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+<head>
+    <link rel="alternate" type="application/atom+xml" href="/feed.atom">
```

Comment:
```suggestion
    <link rel="alternate" type="application/atom+xml" href="/feed.atom">
    <link rel="stylesheet" href="/main.css">
```

This page is missing its main.css stylesheet. With this inclusion, you can then remove the .title and .small-title css written further down this file.

Reply:
While rate limited, all requests to load additional content will be blocked. I have chosen to manually include the styles because of that.